scripts/create_newcase

```diff
@@ -154,6 +154,9 @@ def parse_command_line(args, cimeroot, description):
                         "\nIf provided, the contents of these files will be appended to"
                         "\nthe standard machine files (and any files in ~/.cime).")
 
+    if model == "e3sm":
```

Comment:
Don't restrict it to e3sm.  cesm users might find this useful .  As long as its not a required argument.